### ISSUE TITLE
[Bug]: leading slash at beginning of resource path causing selector to throw 'invalid file name' error at live preview

### ISSUE THREAD

**Issue by Ebbez** [vscode-restructuredtext/vscode-restructuredtext#390]
### Expected behavior

I expected a live preview to appear when pressing the shortcuts to open live preview windows.

### Actual behavior

The live preview window threw an `RST extension got invalid file name: /c:/Users/<REDACTED>/Documents/Projects/<REDACTED>/docs/source/api/v1/reference.rst` error, without any apparent errors in the outputs of either Esbonio or reStructuredText extension. When removing the leading slash, the file does exist and opens up in VS Code (when the path is entered in the address bar of File Explorer).

### Detailed steps

The steps to receive the same result error as me are:

1. Install the reStructuredText according to the official documentation (installing both doc8 & rstcheck, and also Esbonio and docutils)

>  (everything underneath is what I did, but I don't think it is the cause of the error)

2. Make sphinx folder with source and build files separated
3. Set Esbonio's `conf` and `src` folders to `${workspaceFolder}/source` in workspace settings
4. Open live preview with one of the shortcuts (ctrl+k ctrl+r)

### Extension information

188.0.0

### Visual Studio Code information

1.67.1

### Operating system information

Windows 10

### Python information

3.10.4

### Sphinx information

4.5.0

### Contents of conf.py

```python
# Configuration file for the Sphinx documentation builder.
#
# This file only contains a selection of the most common options. For a full
# list see the documentation:
# https://www.sphinx-doc.org/en/master/usage/configuration.html

# -- Path setup --------------------------------------------------------------

# If extensions (or modules to document with autodoc) are in another directory,
# add these directories to sys.path here. If the directory is relative to the
# documentation root, use os.path.abspath to make it absolute, like shown here.
#
# import os
# import sys
# sys.path.insert(0, os.path.abspath('.'))

# -- Project information -----------------------------------------------------

project = '<REDACTED>'
copyright = '2022, <REDACTED>'
author = '<REDACTED>'


# -- General configuration ---------------------------------------------------

# Add any Sphinx extension module names here, as strings. They can be
# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
# ones.
extensions = [
    'sphinxcontrib.httpdomain'
]

# Add any paths that contain templates here, relative to this directory.
templates_path = ['_templates']

# List of patterns, relative to source directory, that match files and
# directories to ignore when looking for source files.
# This pattern also affects html_static_path and html_extra_path.
exclude_patterns = []


# -- Options for HTML output -------------------------------------------------

# The theme to use for HTML and HTML Help pages.  See the documentation for
# a list of builtin themes.
#
html_theme = 'sphinx_rtd_theme'

# Add any paths that contain custom static files (such as style sheets) here,
# relative to this directory. They are copied after the builtin static files,
# so a file named "default.css" will overwrite the builtin "default.css".
html_static_path = ['_static']
```

### Relevant log output

```shell
None seen, files in _build folder are perfectly viewable when opening in Firefox, with the build files also being updated on saves.
```

**Comment by Ebbez** [comment]
Can also confirm that it is due to the 188.0.0 update. When reverting back to 187.0.0 version of the extension it runs perfectly fine.

**Comment by Ebbez** [comment]
The 187.0.0 version of this extension does seem to work with the same configuration, so it seems something breaking has happened, maybe regarding the commit resolving #362.

**Comment by lextm** [comment]
Just published 189.0.0 with a few changes reverted. Please test again once it's live.

**Comment by Ebbez** [comment]
Version 189.0.0 works again, thanks.
